contrib: media_vault use scriptdir (#2689)
bareos/bareos: contrib/misc/media_vault/README.md

Reviewer: Should this section mention the fact that one needs to edit media_vault.sh ?

Author: Another good catch.

Reviewer: This should probably mention the need to edit `media_vault.sh` if a different path is chosen.

Author: Good Idea

python3 -m pip install python_bareos ConfigArgParse
```

.. note:

>    Remind to adjust path used if different from default by editing the helper script.

Reviewer: ```suggestion
   The helper script `media_vault.sh` assumes that the venv is stored in the path used in the example above.
   If you choose a different path, then you need to update `media_vault.sh` to reflect that.
```

dnf install python3-bareos python313-configargparse.noarch
```

> [!IMPORTANT]

> > Remind to remove the call to venv by editing the helper script.

Reviewer: ```suggestion
> Remember to remove the call to `venv` in the helper script `media_vault.sh`.
```

Author: Merged (and rebased)
